modules/rosa-rhods.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * adding_service_cluster/rosa-available-services.adoc
+:_content-type: CONCEPT
+[id="rosa-rhods_{context}"]
+= Red Hat OpenShift Data Science
+
+{rhods} (RHODS) enables users to integrate data and AI and machine learning software to run end-to-end machine learning workflows. It provides a collection of notebook images with the tools and libraries required to develop and deploy data models. This allows data scientists to easily develop data models, integrate models into applications, and deploy applications using Red Hat OpenShift. RHODS is available as an add-on to Red Hat managed environments such as Red Hat {osd-short} and {product-title} (ROSA).
```

Comment:
Not critical, but since you have an {osd} attribute defined that is the full name, instead of adding "Red Hat" in front.

Adding suggestion mode, but please DO NOT click the "Commit suggestion" button to add it to your PR :slightly_smiling_face: 
```suggestion
{rhods} (RHODS) enables users to integrate data and AI and machine learning software to run end-to-end machine learning workflows. It provides a collection of notebook images with the tools and libraries required to develop and deploy data models. This allows data scientists to easily develop data models, integrate models into applications, and deploy applications using Red Hat OpenShift. RHODS is available as an add-on to Red Hat managed environments such as {osd} and {product-title} (ROSA).
```